server/schedule/operator/operator.go

```diff
@@ -53,6 +53,8 @@ type Operator struct {
 	Counters         []prometheus.Counter
 	FinishedCounters []prometheus.Counter
 	AdditionalInfos  map[string]string
+	expect           int // region expect count
```

Comment:
Why the expected region count belongs to an operator?

Reply:
the region that lost majority peers will be putted to priority queue

Reply:
Perhaps this attribute belongs to region, not operator